packages/fxa-event-broker/src/queueworker/queueworker.service.ts

```diff
@@ -132,14 +152,15 @@ export class QueueworkerService
     this.metrics.increment('message.type', { eventType });
     const clientIds = await this.firestore.fetchClientIds(message.uid);
     for (const clientId of clientIds) {
-      const topicName = this.topicPrefix + clientId;
-      const messageId = await this.pubsub.topic(topicName).publishJSON({
+      const messageId = await this.publishMessage(clientId, {
         changeTime: message.timestamp ? message.timestamp : message.ts * 1000,
         event: message.event,
         timestamp: Date.now(),
         uid: message.uid,
+        //@ts-ignore
```

Comment:
What is this ignoring? Seems like it shouldn't be needed?

Reply:
it's because the other dto types don't have `email` and I didn't feel like asserting the type just to put it in a plain json obj.